lib/gollum/app.rb

```diff
@@ -144,6 +144,14 @@ class App < Sinatra::Base
       end
     end
 
+    get %r{\/(edit|create)\/custom.(js|css)} do
+      forbid
+    end
+
+    post %r{\/(deleteFile|rename|edit|revert|create)\/custom.(js|css)(\/.*)?} do
```

Comment:
Because of the `(\/.*)?` at the end, which is needed to match the `revert` route, this regex also blocks e.g. `edit/custom.css/Foo.md`. A bit of an odd usecase, but perhaps still sloppy?